content/tutorial-checks.md

```diff
@@ -0,0 +1,588 @@
+# Checks Tutorial (draft)
+
+***TODO: This whole page is just example requests and responses for now, not a finished page.***
+
+
+<!--{# Just the requests and responses right now #}-->
+
+<!--{# Accounts used in this example:
+rUn84CUYbNjRoTQ6mSW7BVJPSVJNLb1QLo snkuWqxoqt6aeykTbkEWrTMJHrWGM (as the sender)
+rfkE1aSy9G8Upk4JssnwBxhEv5p4mn2KTy sn2Zh1tRZyodU9qNy9tMnQr9UbBss (as the dest.)
```

Comment:
I wonder if the examples might benefit from account identifiers that have a recognizable component, at least for english speakers.  For example I ran `wallet_propose` a few times and stumbled upon

- "rpxQkPYe8ZU7XcYhAzWA2GyzQcpxf4SLED" (master seed "shY8V9rW1NmesV8toKCAGp3g3jXDU")
- "rwvPiear9CeW6J3urr9jogkn8YmVhgRYE (master seed "ssaPf741VokEA4vvzrRbvexbme3P6")
- "rnhLYqjPiNue4dPFJZ8x8rLmVuRoq6Uxrp" (master seed "shcWxfSxJYEA4j3i88E3qWuXGM8YE")

Better examples could be found with some persistence.

I'm just thinking it might make the examples a bit easier to read by distinguishing between "*SLED" and "*RYE".  Just for your consideration.